WordPress/src/androidTest/java/org/wordpress/android/README.md

```diff
@@ -0,0 +1,45 @@
+## UI tests Overview
+
+WordPress for Android has UI acceptance tests for critical user flows through the app, such as login, 
+signup, and publishing. The tests use mocked network requests with [WireMock](http://wiremock.org/), 
+defined in [WordPressMocks](https://github.com/wordpress-mobile/WordPressMocks).
+
+## Running the tests 
+
+**Note that due to the mock server setup, tests cannot be run on physical devices right now.**
+
+
+1. Follow the [build instructions](https://github.com/wordpress-mobile/WordPress-Android#build-instructions)
+ (steps 1-7) to clone, build and run the project in Android Studio.
+2. Run the tests in `Tests in org.wordpress.android.e2e`
+
+There's no additional steps needed to make the tests run against the mock server -- that's configured set up by default
+
+## Adding a new test?
+
+Great! When adding a new UI test, consider:
+
+* Whether you need to test a user flow (to accomplish a task or goal) or a specific feature (e.g. boundary testing).
+* What screens/pages are being tested(defined in `pages/`).
+* What user flows are being used(defined in `flows/`).
+* Any specific UI components being interacted with?(defined in `components/`).
+* What network requests are made during the test (defined in the `WordPressMocks` repo).
```

Comment:
Since the mocks are included in `libs/mocks/WordPressMocks/` what do you think of referring to that here instead of the `WordPressMocks` repo?

Reply:
🤔yeah that'd make more sense to link to it here as well. Something like this? 
```suggestion
* What network requests are made during the test (defined in `libs/mocks/WordPressMocks`)).
```